src/App.tsx

```diff
@@ -15,71 +16,70 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [selectedGood, setSelectedGood] = useState(goods[8]);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const clear = () => setSelectedGood('');
```

Comment:
Use more meaningful names 
```suggestion
  const clearSelection = () => setSelectedGood('');
```